ops-bedrock/docker-compose.yml

```diff
@@ -121,7 +121,7 @@ services:
       OP_BATCHER_L1_ETH_RPC: http://l1:8545
       OP_BATCHER_L2_ETH_RPC: http://l2:8545
       OP_BATCHER_ROLLUP_RPC: http://op-node:8545
-      OFFLINE_GAS_ESTIMATION: false
+      OFFLINE_GAS_ESTIMATION: "false"
```

Comment:
Can you delete this env var? It's not used anymore.

Reply:
yup